docs/cmd/kn_broker_describe.md

```diff
@@ -33,7 +33,7 @@ kn broker describe NAME
       --allow-missing-template-keys   If true, ignore any errors in templates when a field or map key is missing in the template. Only applies to golang and jsonpath output formats. (default true)
   -h, --help                          help for describe
   -n, --namespace string              Specify the namespace to operate in.
-  -o, --output string                 Output format. One of: json|yaml|name|go-template|go-template-file|template|templatefile|jsonpath|jsonpath-file|url.
+  -o, --output string                 Output format. One of: json|yaml|name|go-template|go-template-file|template|templatefile|jsonpath|jsonpath-as-json|jsonpath-file|url.
```

Comment:
Should we then make `jsonpath-as-files but maybe this should be in different PR

Reply:
@maximilien this is coming directly from a dependency, we delegate all option handling to Kubernetes-client. The name comes from there, so not so much we can change here (and we also shouldn't do it if we could in order to avoid surprises to Kubernetes users)